src/graph/graph-state.module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from '@nestjs/common';
+import { GraphStateManager } from './graph-state-manager';
+import { createGraphStateManager } from './graph-state-manager.factory';
+import { EnvironmentType } from '@dsnp/graph-sdk';
+
+@Module({
+    imports: [],
+    providers: [
+        {
+            provide: GraphStateManager,
+            useFactory: () => {
+                const environmentConfig = {
+                    environmentType: EnvironmentType.Mainnet // Or any other environment
```

Comment:
Would be good to take this from the environment somehow

Reply:
Yep will move it to configs